### PR TITLE
Initial Import: S6 and run.d integration

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: container/root/run.d/.gitkeep [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (bossjones) [inline container/root/run.d/.gitkeep]: Just double checking, this file is suppose to be a bash script? I thought `.gitkeep` is just for making sure blank directories get committed?

**Comment by bossjones** [comment]
:+1:

**Comment by mannytoledo** [comment]
:+1:

**Comment by mannytoledo** [comment]
Where's the alpine version :trollface:

**Pull request merged**